test/integration/crud/explain.test.ts

```diff
@@ -109,26 +109,21 @@ describe('CRUD API explain option', function () {
           } else {
             explainDocument = response;
           }
+          const explainJson = JSON.stringify(explainDocument);
```

Comment:
Can we get rid of the if-statement above?  We're always checking the stringified JSON for the presence of `queryPlanner` so the exact location doesn't matter.

Reply:
Updated.